guide/lib/helpers/formatters.rb

```diff
@@ -33,9 +33,9 @@ def format_haml(raw, data = nil)
                  {}
                end
 
-      template = Haml::Engine.new(raw).render(FakeView.new, **locals)
+      template = Tilt::HamlTemplate.new { raw }.render(FakeView.new, **locals)
```

Comment:
do you have some input on this change?

Reply:
Yes. There's a bug with Haml5 + nanoc which means that our example code:

```haml
= render('/partials/example.*') do
  %h1 Some content
```

was double-rendered. It outputs the "Some content" bit *and* the example (with "Some content" in it). I tried a bunch of hacks and ended up finding a way with Haml6 and that bug report in nanoc.

I'll have a look again and see if I can make it happen with Haml5 and less changes, I've spent hours on this problem so I might have better luck this time.

Reply:
👌 fine with me, you don't have to spend more time on this